modules/apps/29-fee/keeper/grpc_query_test.go

```diff
@@ -0,0 +1,165 @@
+package keeper_test
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/query"
+	"github.com/cosmos/ibc-go/modules/apps/29-fee/types"
+	channeltypes "github.com/cosmos/ibc-go/modules/core/04-channel/types"
+)
+
+var (
+	expPackets []*types.IdentifiedPacketFee
+	refundAcc  sdk.AccAddress
+	ackFee     sdk.Coins
+	receiveFee sdk.Coins
+	timeoutFee sdk.Coins
+)
```

Comment:
These should be kept within scope of the test function they are used in (otherwise leftover assignments could unintentionally be used between tests)